docs/definition.rst

```diff
@@ -78,13 +107,20 @@ Containerfile, so they will persist if ``podman build`` is called manually.
 If the same variable is specified in the CLI ``--build-arg`` flag,
 the CLI value will take higher precedence.
 
+.. _ansible_config:
+
 ansible_config
 ^^^^^^^^^^^^^^
 
 When using an ``ansible.cfg`` file to pass a token and other settings for a
 private account to an Automation Hub server, listing the config file path here
-(as a string) will enable it to be included as a build argument in the initial
-phase of the build.
+(as a string) will enable it to be use for the Galaxy installation build phase.
```

Comment:
I would really love to kill this whole thing as well...

Reply:
Would have to be in a v3 schema since we released 1.2 with support for this in the v2 schema.